atlp-rwanda/e-commerce-ones-and-zeroes-fn
# User adding item in cart

## 1.What does this PR do?
Users should be able to add product in the cart
# ⚠️⚠️How to test this PR
1. Clone `develop` branch and checkout to `user-adding-item-in-cart`
2. Install all required dependencies `npm install`
3. Follow `.env.example` to create your `.env` file
4. Start the application using `npm start`
5.  Navigate to login and start by signing up, verify your account 
![Screenshot 2024-07-06 222043](https://github.com/atlp-rwanda/e-commerce-ones-and-zeroes-fn/assets/57124925/a6f70ac4-ce4c-4f7b-8775-78cce9916599)
